mainlyetcetera/quick-quotes
# Feature/render header

## Is this a fix or a feature?
- feature
## What is the change/fix?
- clean up default `create-react-app` files
- add `Header` component
- render `Header` in `App`
- make `Header` grey
## How should this be tested?
- open `localhost:3000`
- observe grey header
- observe app title
- observe welcome msg
## This PR is related to which issues:
Closes #12
